app-backend/db/src/main/resources/migrations/V72__add_index_to_is_active_on_annotation_project.sql

```diff
@@ -0,0 +1,4 @@
+-- add btree index on is_active field of annotation_projects table
+CREATE INDEX IF NOT EXISTS annotation_projects_is_active_idx
+ON public.annotation_projects
+USING btree (is_active);
```

Comment:
In the spirit of azavea/raster-foundry-platform#1098, it might make sense eventually to capture this as another status like `DELETING` instead of `is_active`, since `active` doesn't capture the specifics as precisely as the status would. But I don't think that's important to do now, since `is_active` is consistent with how we model this elsewhere and at least for now it _only_ means deleting / not deleting.